frontends/web/src/components/settingsButton/settingsToggle.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2023 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ReactNode } from 'react';
+import { Toggle, TToggleProps } from '../toggle/toggle';
+import style from './settingsToggle.module.css';
+
+type Props = TToggleProps & {
+  children: ReactNode;
+};
+
+export const SettingsToggle = ({
+  children,
+  ...props
+}: Props) => {
+  return (
+    <div className={style.setting}>
+      {children}
+      <Toggle {...props} />
+    </div>
+  );
+};
```

Comment:
DOM is slightly different and uses less nested elements, take a look at settings.tsx